### ISSUE TITLE
Please read :(

### ISSUE THREAD

**Issue by Nathaniel-428** [TotalFreedomMC/TotalFreedomMod#211]
# Please read

So, I heard you're in need of new developers.

If I'm going to be honest I have not been kind to TF or it's community

### Let's make a deal:

I will help you develop TFM, but you really gotta stop banning me.

So, Nathaniel_428 for developer?

Please, I really do wanna join. This is a one in a lifetime opportunity I want to take too.

**Comment by ncurran02** [comment]
Can you like not?

**Comment by Nathaniel-428** [comment]
:(

**Comment by ncurran02** [comment]
No.

**Comment by VideoGameSmash12** [comment]
We have told you several times to go away and quit harassing the community, however you keep coming back. I'm going to say this again: move on from the server. You are not welcome here anymore.

In other words: No.

**Comment by gomalley411** [comment]
I’m gonna be as nice as I can here, but seeing as you are getting on everyone’s LAST NERVE here, please for the love of God just fuck off. I never thought I would be saying that to an OP, but not only have you been pushing my buttons so hard with how you handled getting permbanned, but also we’ve told you several times you are no longer welcome in our community yet you still try and ignore that. This is borderline sexual harassment/stalking and I am seriously considering getting your local authorities involved in this as well if you don’t stop. 

Please stay away from TotalFreedom and the entire community in any and all ways, shapes or forms. That means don’t bypass, don’t register new forum accounts to try and circumvent your forum ban, don’t send mass emails to us and especially don’t contact bowie again. If you don’t respect our wishes this time around, then I will have no choice but to call your local police. And we both know you wouldn’t like that. 

So in other words, NO.

Gommeh

Sent from my iPhone



George O’Malley

**Comment by squareGITHUB** [comment]
we could use a /nathan command you have been added to co-owner and unbanned

**Comment by Nathaniel-428** [comment]
>>
> I am seriously considering getting your local authorities involved

That will backfire.

>>
> If you don’t respect our wishes

You never wished for anything.

**Comment by script-head** [comment]
Looking at your fork, I really don't want you on the team.
